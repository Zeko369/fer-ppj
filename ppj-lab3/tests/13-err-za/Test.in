<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 c
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 123
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 x
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 5
       <T_lista>
        $
      <E_lista>
       $
   <lista_naredbi>
    <naredba>
     <za_petlja>
      KR_ZA 3 za
      IDN 3 c
      KR_OD 3 od
      <E>
       <T>
        <P>
         IDN 3 c
        <T_lista>
         OP_PUTA 3 *
         <T>
          <P>
           BROJ 3 2
          <T_lista>
           $
       <E_lista>
        $
      KR_DO 3 do
      <E>
       <T>
        <P>
         IDN 3 c
        <T_lista>
         OP_PUTA 3 *
         <T>
          <P>
           BROJ 3 3
          <T_lista>
           $
       <E_lista>
        $
      <lista_naredbi>
       <naredba>
        <naredba_pridruzivanja>
         IDN 4 x
         OP_PRIDRUZI 4 =
         <E>
          <T>
           <P>
            IDN 4 x
           <T_lista>
            $
          <E_lista>
           OP_PLUS 4 +
           <E>
            <T>
             <P>
              IDN 4 c
             <T_lista>
              $
            <E_lista>
             $
       <lista_naredbi>
        $
      KR_AZ 5 az
    <lista_naredbi>
     $
