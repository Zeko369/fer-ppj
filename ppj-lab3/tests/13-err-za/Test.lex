IDN 1 c
OP_PRIDRUZI 1 =
BROJ 1 123
IDN 2 x
OP_PRIDRUZI 2 =
BROJ 2 5
KR_ZA 3 za
IDN 3 c
KR_OD 3 od
IDN 3 c
OP_PUTA 3 *
BROJ 3 2
KR_DO 3 do
IDN 3 c
OP_PUTA 3 *
BROJ 3 3
IDN 4 x
OP_PRIDRUZI 4 =
IDN 4 x
OP_PLUS 4 +
IDN 4 c
KR_AZ 5 az
