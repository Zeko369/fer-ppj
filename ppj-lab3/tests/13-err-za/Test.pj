c = 123
x = 5
za c od c * 2 do c * 3
 x = x + c
az
