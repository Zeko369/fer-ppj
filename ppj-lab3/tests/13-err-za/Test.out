err 3 c
